state.py

```diff
@@ -29,6 +29,18 @@ def eval(self):
         for y,line in enumerate(self.occupied):
             if any(line): return y
         return consts.HEIGHT
+    
+    def eval_place(self):
+        consts.WIDTH = 4
+
```

Comment:
Good test, but put this in the test_eval_place.py file too since it is for testing only (also recommend to name it test_place.py, test_eval.py was called that because it tested the eval function)